conf/routes

```diff
@@ -42,6 +42,8 @@ POST    /reindex-publish                com.gu.atom.play.ReindexController.newPu
 GET     /reindex-preview                com.gu.atom.play.ReindexController.previewReindexJobStatus()
 GET     /reindex-publish                com.gu.atom.play.ReindexController.publishedReindexJobStatus()
 
+POST    /pluto/resend/:id               controllers.PlutoController.resendAtomMessage(id)
```

Comment:
can this be `/api2/pluto/resend/:id` to keep [the namespace](https://github.com/guardian/media-atom-maker/blob/master/conf/routes#L17-L19)